bindings/xen/console.c

```diff
@@ -102,13 +102,15 @@ void console_write(const char *buf, size_t len)
     if (!console_initialised)
         return;
 
-    size_t written = 0;
+    size_t written = 0, count = 0;
     XENCONS_RING_IDX cons = LOAD_ACQUIRE(&console_ring->out_cons);
 
     do
     {
-        /* Try and put some data into the ring. */
-        written = console_write_some(&buf[written], len - written);
```

Comment:
Perhaps this was supposed to be:
```c
written += console_write_some(&buf[written], len - written);
```

Reply:
Of course! It's really simpler, thanks :)